### PR TITLE
GPC-185: Create baseline migration

### PR DESCRIPTION
**Pull request by CarlyG55**

### CODE REVIEW

File: src/main/resources/db/baseline/B1_29__baseline.sql [hunk 1]
```diff
@@ -0,0 +1,86 @@
+CREATE TABLE supplier
```
Comment (ethanmills) [inline src/main/resources/db/baseline/B1_29__baseline.sql:1]: Do the B prefixes work? I thought you needed to pay for flyway enterprise to get that

File: src/main/resources/db/devonly/R__dev_data.sql [hunk 2]
```diff
@@ -30,36 +30,14 @@ Begin
 End;
 $$;
 
-DELETE
```
Comment (oskarwilliams378) [inline src/main/resources/db/devonly/R__dev_data.sql:33]: These are to make the migration repeatable, everything added should be removed

File: src/main/resources/db/migration/V1__baseline.sql [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    id                       UUID PRIMARY KEY         NOT NULL DEFAULT gen_random_uuid(),
+    acquirer_subscription_id UUID                     NOT NULL,
+    data_id                  VARCHAR(80)              NOT NULL,
+    when_created             TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+    event_time               TIMESTAMP WITH TIME ZONE NOT NULL DEFAULT now(),
+    deleted_at               TIMESTAMP WITH TIME ZONE,
+
+    CONSTRAINT fk_acquirer_subscription
+        FOREIGN KEY (acquirer_subscription_id)
+            REFERENCES acquirer_subscription
```
Comment (oskarwilliams378) [inline src/main/resources/db/migration/V1__baseline.sql:56]: does this need this?
```suggestion
            REFERENCES acquirer_subscription (id)
```

File: src/main/resources/db/migration/V1__baseline.sql [hunk 4]
```diff
@@ -0,0 +1,86 @@
+CREATE TABLE supplier
```
Comment (oskarwilliams378) [inline src/main/resources/db/migration/V1__baseline.sql:1]: could we sort these creations a little more, either by type or by alphabet (mainly thinking the `acquirer_subscription_enrichment_field` should be after the `acquirer_subscription`)

**Review by oskarwilliams378 — CHANGES_REQUESTED** [review]

**Review by oskarwilliams378 — APPROVED** [review]

**Pull request merged**